doc/strftime_formatting.rdoc

```diff
@@ -461,3 +461,86 @@ The integer width specifier gives a minimum width for the returned string:
   Time.new(2002, 12).strftime('%B')   # => "December" # No width specifier.
   Time.new(2002, 12).strftime('%10B') # => "  December"
   Time.new(2002, 12).strftime('%3B')  # => "December" # Ignored if too small.
+
+== Specialized Format Strings
+
+Here are a few specialized format strings,
+each based on an external standard.
+
+=== HTTP Format
+
+The HTTP date format is based on
+{RFC 2616}[https://datatracker.ietf.org/doc/html/rfc2616],
+and treats dates in the format <tt>'%a, %d %b %Y %T GMT'</tt>;
+see {Formats for Dates and Times}[https://docs.ruby-lang.org/en/master/strftime_formatting_rdoc.html]:
+
+  d = Date.new(2001, 2, 3) # => #<Date: 2001-02-03>
+  # Return HTTP-formatted string.
+  httpdate = d.httpdate    # => "Sat, 03 Feb 2001 00:00:00 GMT"
+  # Return new date parsed from HTTP-formatted string.
+  Date.httpdate(httpdate)  # => #<Date: 2001-02-03>
+  # Return hash parsed from HTTP-formatted string.
+  Date._httpdate(httpdate)
+  # => {:wday=>6, :mday=>3, :mon=>2, :year=>2001, :hour=>0, :min=>0, :sec=>0, :zone=>"GMT", :offset=>0}
+
+=== ISO 8601 Format
```

Comment:
I think we should collate the "ISO 8601 Format Specifications" section with this so that we don't have info about ISO 8601 scattered throughout the page.

Reply:
Section "ISO 8601 Format" removed; already covered in section "ISO 8601 Format Specifications."